### PR TITLE
loopd: nil-pointer bug when showing subsystems

### PR DESCRIPTION
**Pull request by ziggie1984**
Just a short drive-bug fix: loopd crashes when you wanna show the logging subsystems with `loopd --debuglevel show`

This fix just copies the rogue code to another place, but I think a better fix would be to get rid of the global pointer and maybe just return the logger when setting it up ?

#### Pull Request Checklist
- [x] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

### CODE REVIEW

File: loopd/run.go [hunk 1]
```diff
@@ -204,6 +197,13 @@ func Run(rpcCfg RPCConfig) error {
 	logWriter := build.NewRotatingLogWriter()
 	SetupLoggers(logWriter, shutdownInterceptor)
 
+	// Special show command to list supported subsystems and exit.
+	if config.DebugLevel == "show" {
```
Comment (GeorgeTsagk) [inline loopd/run.go:201]: Is this fix effective because `if err := Validate(&config)` precedes now?

Comment (ziggie1984) [reply loopd/run.go:201]: hmm let me check

Comment (ziggie1984) [reply loopd/run.go:201]: yeah so debuglevel validation seems not part and not necessary to validate so it should be fixed

**Review by hieblmi — APPROVED** [review]
Rebased this PR and tested, as @ziggie1984 mentioned the validation doesn't check the debuglevel, so moving the code is fine. LGTM 💾

✅ tAck

**Pull request merged**